scripts/core/editor2/editor.js

```diff
@@ -698,7 +698,8 @@ angular.module('superdesk.apps.editor2', [
     'superdesk.apps.editor2.utils',
     'superdesk.apps.spellcheck',
     'superdesk.apps.authoring',
-    'angular-embed'
+    'angular-embed',
+    'angular-embed.handlers'
```

Comment:
This seems unusual. Should `angular-embed` not automatically include `angular-embed.handlers`?